tests/exchange/test_exchange.py

```diff
@@ -173,35 +173,74 @@ def test_validate_order_time_in_force(default_conf, mocker, caplog):
     ex.validate_order_time_in_force(tif2)
 
 
-def test_symbol_amount_prec(default_conf, mocker):
+@pytest.mark.parametrize("amount,precision_mode,precision,expected", [
```

Comment:
I did add a few more tests ... I think for amount we're clear (it should truncate - thats how it's always been) - however it should support the "new" method as well.

I've also added the tests for price_to_precision - but these may be wrong depending if we decide to "round up" ...